daemon/mgr/container.go

```diff
@@ -280,6 +281,13 @@ func (mgr *ContainerManager) Restore(ctx context.Context) error {
 			return err
 		}
 
+		// recover the running or paused container.
+		if !(c.IsRunning() || c.IsPaused()) {
```

Comment:
running or paused containers don't to recover?

Reply:
no need to recover the created/stopped container